plugins/chat/app/models/concerns/chat/mappable.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Chat
+  module Mappable
```

Comment:
We should find a more descriptive name, anything can be "Mappable" maybe, `RemappablePolymorphism` or something like this...